### ISSUE TITLE
passing router options

### ISSUE THREAD

**Issue by joprice** [Daydreamer-riri/vite-react-ssg#45]
I'm seeing warnings like the following

```
 React Router Future Flag Warning: Casing of `formMethod` fields is being normalized to uppercase in v7. You can use the `v7_normalizeFormMethod` future flag to opt-in early. For more information, see https://reactrouter.com/v6/upgrading/future#v7_normalizeformmethod.
```

Is is possible to pass such options to the router via a vite config or via a provider?

**Comment by Daydreamer-riri** [comment]
Hi @joprice, This feature has been implemented in 0.8.2!

**Comment by joprice** [comment]
Finally got around to using this and it works great. Thanks!
